hathor/websocket/factory.py

```diff
@@ -197,7 +197,10 @@ def serialize_message_data(self, event, args):
             data = tx.to_json_extended()
             data['is_block'] = tx.is_block
             # needed to check if the transaction is from the mempool
-            data['first_block'] = tx.get_metadata().first_block
+            first_block = tx.get_metadata().first_block
```

Comment:
We are changing the type of `first_block` along the code. Maybe rename this to `first_block_bytes` or the other to `first_block_hex`?

Reply:
Do you mean rename the JSON property `'first_block'` to `'first_block_hex'`? I don't think that's good because on all API surfaces we already use hex for all fields that contain a hash without suffixing it with `_hex`, like `'hash'`, `'tx_id'`, `'parents'`, ..., and on the other hand, we don't use `_bytes` on the "internal" code (not API surface). So I think this is best as is.

This change in "type" also doesn't affect any API because this `data` was never able to be serialized while that fields was using `bytes`. This is actually a bug that has to be fixed in order for this API to start working again.

What do you think? @msbrogli @pedroferreira1

Reply:
I think it's only the variable name.

`first_block = tx.get_metadata().first_block` and then you `first_block = first_block.hex()`

Reply:
Makes sense, I'll rename that variable.

Reply:
What do you think?

Reply:
Perfect for me

Reply:
It was just the variable name. Thanks!